DDDEastAnglia.Tests/Sponsors/sponsor_list/context.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using DDDEastAnglia.Controllers;
+using DDDEastAnglia.DataAccess;
+using DDDEastAnglia.Models;
+
+namespace DDDEastAnglia.Tests.Sponsors.sponsor_list
+{
+    public class context
```

Comment:
didn't know they had a name.. it's my new default for testing.. glad you like it.